classes/db/Db.php

```diff
@@ -704,6 +706,11 @@ public function getValue($sql, $use_cache = true)
 
     /**
      * Get number of rows for last result.
+     * Must be used solely with select queries.
+     * Results are unreliable with DbPDO (https://www.php.net/manual/en/pdostatement.rowcount.php)
+     * With Db Caching enabled, it works only after `executeS()` and `getRow()`
+     *
+     * @deprecated Never used throughout current codebase
```

Comment:
It is used:

![Screenshot 2021-08-20 at 14 38 31](https://user-images.githubusercontent.com/1009343/130234358-d0ee87d6-98af-411d-ad71-969290ef70d6.png)

Reply:
@eternoendless
I don't know how I searched then. I think best is to close the PR and the issue.